### DISCUSSION TITLE
Any way to activate the crt filter by default on latest version ?

### DISCUSSION THREAD

**Discussion by Darenn** [nesbox/TIC-80#1845] [Q&A] [answered]
As an artistic choice, I would like to activate the crt filter by default in my game.
I still want the player to be able to deactivate it in the menu.

I can upload win/linux versions with crt activated by default, just by running the cartridge and activating the option in the menu. I imagine it's saving the preferences somewhere on the cartridge.

It seems I can't do the same with the html version.

Any way to do that in html ?

Thanks!

**Comment by nesbox** [comment, marked as answer]
You can use `tic80 --crt` command line parameter, also try to edit `index.html` in the exported .zip to enable CRT in HTML 
``` js
var Module = {canvas: document.getElementById('canvas'), arguments:['cart.tic', '--crt']}
```

**Reply by Darenn** [reply]
Works perfectly, thanks!

**Comment by lustfeind** [comment]
this isnt working anymore?

**Reply by farvardin** [reply]
@lustfeind I've just compiled it with "cmake -DBUILD_SDLGPU=On" option, and the tic80 --crt option is working for me. Does crt works for you if you type F6 on the tic80 console?

**Reply by lustfeind** [reply]
Yes, F6 is working and I compiled it with DBUILD_SDLGPU=On. 
But I can't manage to start with crt on. 

```
 <script type="module">
        import startTic80 from './tic80.js'

        const initialize = () => {
            const canvasSelector = '#canvas'
            const canvasElement = document.querySelector(canvasSelector)
            const playElement = document.getElementById('tic80-play')

            const options = {
                canvas: canvasElement,

                arguments: ['cart.tic', '--crt'], 
.
.
.

```

**Reply by NNNN618** [reply]
> this isnt working anymore?

same issue here, do you find a solution? thx
